src/helpers/report.cjs

```diff
@@ -23,8 +23,8 @@ const reportMemberPriority = [
 	'gitSha',
 	'name',
 	'status',
-	'lmsInstance',
-	'lmsBuild',
+	'lmsBuildNumber',
```

Comment:
does it matter that the order of these things got swapped?

Reply:
It does but I swapped them on purpose to match some other documentation. This just determined the order of the elements when serialized as JSON output.